bpf/tests/prepend_name_test.c

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/* Copyright Authors of Cilium */
+
+//go:build ignore
```

Comment:
Question: Why do we need that?

Reply:
So this is a bit weird but you can't have a `.c` file in the same directory as a go module unless you are using CGO 🤔  most Go commands will return `C source files not allowed when not using cgo`.

Reply:
I wanted to house the C file with the BPF test harness and the Go file in the same directory so it's needed!